projects/js-packages/publicize-components/src/hooks/use-post-meta/index.js

```diff
@@ -18,7 +23,7 @@ export function usePostMeta() {
 
 			const isPublicizeEnabled = meta.jetpack_publicize_feature_enabled ?? true;
 			const jetpackSocialOptions = meta.jetpack_social_options || {};
-			const attachedMedia = jetpackSocialOptions.attached_media || [];
+			const attachedMedia = jetpackSocialOptions.attached_media || DEFAULT_ATTACHED_MEDIA;
 			const imageGeneratorSettings = jetpackSocialOptions.image_generator_settings ?? {
```

Comment:
Do you think we should have a DEFAULT_IMAGE_GENERATOR_SETTINGS  object too? That way we won't need to remember to list the individual properties in the dependency array.

Should we also have an empty object constant to handel the default for `jetpackSocialOptions`?

Reply:
Done in 57db8c947e4a6b5c99bcbf0ce6be013ac06924c9